rust/lance-index/src/scalar/bitmap.rs

```diff
@@ -715,6 +726,10 @@ impl ScalarIndexPlugin for BitmapIndexPlugin {
         true
     }
 
+    fn index_type(&self) -> IndexType {
+        IndexType::Bitmap
+    }
```

Comment:
Why is this method necessary?

Reply:
This might interact with what was done here: https://github.com/lancedb/lance/pull/5221

Reply:
Yep, I'm thinking about this too, but we need a way to tell which index type it is without loading the index.